.github/workflows/commitlint.yml

```diff
@@ -7,4 +7,4 @@ on:
 
 jobs:
   commitlint:
-    uses: edx/.github/.github/workflows/commitlint.yml@main
+    uses: edx/.github/.github/workflows/commitlint.yml@master
```

Comment:
Q: So this line isn't referring to the `commitlint.yml` file in this repo? This repo has a `main` branch but not `master`.

Reply:
correct. the older repo still uses master